dashboard/test/helpers/levels_helper_test.rb

```diff
@@ -494,6 +494,31 @@ def stub_country(code)
     assert_equal '/s/test_script/stage/2/puzzle/1/page/1', build_script_level_path(stage.script_levels[0], {puzzle_page: '1'})
   end
 
+  test 'build_script_level_path uses names for bonus levels to support cross-environment links' do
+    input_dsl = <<-DSL.gsub(/^\s+/, '')
```

Comment:
you can now replace this with [squiggly heredoc](https://infinum.com/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc)!
```suggestion
    input_dsl = <<~DSL
```
kudos to @islemaster for reminding me about this enough times for me to actually remember to suggest it.

Reply:
ooooh